lib/run.js

```diff
@@ -27,6 +27,8 @@ const probot = createProbot({
  * Start the probot worker.
  */
 function start() {
+  // We are always behind a proxy, but we want the source IP
+  probot.server.set('trust proxy', true);
```

Comment:
What will this do?

Reply:
This will cause express to use the originating IP address from the client instead of the most recent IP address which is that of the internal load balancer.

Reply:
Details: https://expressjs.com/en/guide/behind-proxies.html